File: referralhero/src/main/java/com/sdk/rh/networking/ReferredBy.java
package com.sdk.rh.networking;

public class ReferredBy {

    private String id;
    private String name;
    private String email;
    private String code;
    private Integer peopleReferred;
    private Integer points;

    public String getId() {
        return id != null ? id : "";
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getName() {
        return name != null ? name : "";
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getEmail() {
        return email != null ? email : "";
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getCode() {
        return code != null ? code : "";
    }

    public void setCode(String code) {
        this.code = code;
    }

    public Integer getPeopleReferred() {
        return peopleReferred != null ? peopleReferred : 0;
    }

    public void setPeopleReferred(Integer peopleReferred) {
        this.peopleReferred = peopleReferred;
    }

    public Integer getPoints() {
        return points != null ? points : 0;
    }

    public void setPoints(Integer points) {
        this.points = points;
    }
}

File: referralhero/src/main/java/com/sdk/rh/networking/ReferredBy.kt
package com.sdk.rh.networking

class ReferredBy {
    var id: String? = null
        get() = if (field != null) field else ""
    var name: String? = null
        get() = if (field != null) field else ""
    var email: String? = null
        get() = if (field != null) field else ""
    var code: String? = null
        get() = if (field != null) field else ""
    var peopleReferred: Int? = null
        get() = if (field != null) field else 0
    var points: Int? = null
        get() = if (field != null) field else 0
}